pkg/apis/org/v1/che_types.go

```diff
@@ -544,41 +480,36 @@ type CheClusterSpecStorage struct {
 type CheClusterSpecK8SOnly struct {
 	// Global ingress domain for a K8S cluster. This MUST be explicitly specified: there are no defaults.
 	IngressDomain string `json:"ingressDomain,omitempty"`
-	// Strategy for ingress creation. This can be `multi-host` (host is explicitly provided in ingress),
-	// `single-host` (host is provided, path-based rules) and `default-host.*`(no host is provided, path-based rules).
-	// Defaults to `"multi-host`
-	// Deprecated in favor of "serverExposureStrategy" in the "server" section, which defines this regardless of the cluster type.
-	// If both are defined, `serverExposureStrategy` takes precedence.
+	// Strategy for ingress creation. Options are: `multi-host` (host is explicitly provided in ingress),
+	// `single-host` (host is provided, path-based rules) and `default-host` (no host is provided, path-based rules).
+	// Defaults to `multi-host` Deprecated in favor of `serverExposureStrategy` in the `server` section,
```

Comment:
> Defaults to `multi-host` 
Should this sentence end with point '.' ?

Reply:
To be honest I don't know. I applied suggested changes from https://github.com/tolusha/che-docs/pull/4/commits